Dockerfile_build

```diff
@@ -4,6 +4,9 @@
 # and verification, we can list the rust container as a prior build stage, and
 # then pull in the artifacts we need. There is an added benefit that tagged versions
 # also include minory releases, so 1.2 includes 1.2.1 and so on, for bugfix releases.
+#
+# NOTE: If you change this, you must also change `RUST_VERSION` in the `build-windows`
+# job in `.circleci/config.yml`.
```

Comment:
Thanks for this comment. It's the one thing I was looking for when I saw the review request come in.